cmd/localCI/Makefile

```diff
@@ -0,0 +1,24 @@
+DESTDIR :=
+PREFIX := /usr/local
+BINDIR := $(PREFIX)/bin
+SYSCONFDIR := $(PREFIX)/etc
+LIBEXECDIR := $(PREFIX)/libexec
+
+SOURCES := $(shell find . 2>&1 | grep -E '.*\.(c|h|go)$$')
```

Comment:
It's also not needed, just calling `go build` unconditionally is perfectly fine.